File: teste.cpp
#include <stdio.h>
#include <stdlib.h>

int main(void)
{
    int idade = 0;
    printf("Digite sua idade: ");
    scanf("%d", &idade);
    if(idade >= 18)
    {
		printf("E maior de idade");
    }
    else
    {
		printf("E menor de idade");
    }

    getchar();
    return 0;
}

File: teste.cs
using System;
class Program
{
        static void Main(string[] args)
        {
            Console.WriteLine("Digite sua Idade: ");
	    int idade = int.Parse(Console.ReadLine());
	    
            if(idade >= 18)
	       Console.WriteLine("É maior de idade");
            else
              Console.WriteLine("É menor de idade"); 

	    Console.ReadLine();
        }
}

